.github/workflows/compute-impacted-libraries.yml

```diff
@@ -0,0 +1,63 @@
+name: "Compute libraries impacted by the change"
+
+on:
+  workflow_call:
+    outputs:
+      impacted_libraries:
+        description: "JSON list of libraries impacted by the change"
+        value: ${{ jobs.main.outputs.impacted_libraries }}
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    outputs:
+      impacted_libraries: ${{ steps.compute_impacted_libraries.outputs.result }}
+    steps:
+      - uses: actions/checkout@v4
+      - run:  |
+          git fetch origin ${{ github.event.pull_request.base.sha || github.sha }}
+          git diff --name-only HEAD ${{ github.event.pull_request.base.sha || github.sha }} >> modified_files.txt
+          cat modified_files.txt
+      - name: Compute impacted libraries
+        id: compute_impacted_libraries
+        shell: python
+        run: |
+          import os
+          import json
+          import re
+
+          libraries = "cpp|dotnet|golang|java|nodejs|php|python|ruby"
+          result = set()
+
+          match = re.search(rf"^\[({libraries})\]", os.environ["PR_TITLE"].lower())
```

Comment:
is PR_TITLE variable on os.environment by default?

Reply:
it's provided with then `env` section : https://github.com/DataDog/system-tests/pull/2356/files#diff-e45c08fbf0564cb6b246f7655ad62b78d222c5d3a0232a0790cbe6fd48b243ddR59